pkg/events/error.go

```diff
@@ -27,6 +27,21 @@ type ErrorInfo struct {
 	Custom   log.Marshaler
 }
 
+// LoggableError is the minimum required to make an error loggable.
```

Comment:
This seems less powerful than the existing [events.NewError](https://pkg.go.dev/github.com/getoutreach/gobox@v1.57.3/pkg/events#NewErrorInfo), which already implements `log.Marshaler`.  Could that be used instead?

Reply:
TBH this is overcomplicated, I just wanted a thing that would let me pass an error to status, with minimal fuss. I ended up moving `loggableError` to the trace package where it only does that.

I have no real objection to using events.NewError instead, except that I worry it will make the status in traces hard to read due to its length, and if you use the new trace.Error fn, it will duplicate information available at other points int he span.

Reply:
Fixed